src/renderer/api/endpoints/ingress.api.ts

```diff
@@ -107,6 +108,20 @@ export class Ingress extends KubeObject {
     }
     return ports.join(", ")
   }
+
+  getLoadBalancers() {
+    const ingressAddresses: string[] = [];
+    const { status: { loadBalancer } } = this;
+
+    if (!loadBalancer) return [];
+
+    loadBalancer.ingress.forEach(address => {
```

Comment:
This crashes when there are no external addresses

```suggestion
    loadBalancer.ingress?.forEach(address => {
```

Reply:
Good spot @jim-docker. I guess the only time this can happen though is if an ingress object exists without an ingress controller being present? Still something we should handle